### PR TITLE
Spinning up compiler repo

### PR DESCRIPTION
**Pull request by rsheeter**
Rudimentary rigging for IR w/toml and bincode as text and binary representation respectively. I'm more interested in feedback on the structure than the naming, we can file an issue for the inevitable naming debate :)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/target
+Cargo.lock
+.venv/
+
+.DS_Store
```
Comment (dfrg) [inline .gitignore:5]: ty for this! Need to add to our other .gitignore files too

File: ir/src/ir.rs [hunk 2]
```diff
@@ -0,0 +1,37 @@
+//! Serde types for font IR. See TODO:PublicLink.
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, PartialEq)]
+pub struct Axis<'a> {
+    tag: &'a str,
```
Comment (dfrg) [inline ir/src/ir.rs:7]: I know this is just scaffolding but this should likely be an owned String.

File: ufo2ir/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+license = "MIT/Apache-2.0"
+description = "Converts UFO or UFO+designspace to font ir for compilation."
+repository = "https://github.com/googlefonts/fontations"
+readme = "README.md"
+categories = ["text-processing", "parsing", "graphics"]
+
+[features]
+
+[dependencies]
+font-types = { git = 'https://github.com/googlefonts/fontations' }
```
Comment (dfrg) [inline ufo2ir/Cargo.toml:14]: Shouldn’t need this explicit dep since it’s now re-exported from read-fonts.

**Review by dfrg — APPROVED** [review]
Let’s go!

File: ir/src/ir.rs (hunk 2 above)
Comment (rsheeter) [reply ir/src/ir.rs]: Done

File: ir/Cargo.toml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+[package]
+name = "ir"
+version = "0.0.1"
+edition = "2021"
+license = "MIT/Apache-2.0"
+description = "IR for font compilation."
+repository = "https://github.com/googlefonts/fontations"
```
Comment (anthrotype) [inline ir/Cargo.toml:7]: copy/pasta in this repository URL

File: ir/Cargo.toml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+[package]
+name = "ir"
```
Comment (anthrotype) [inline ir/Cargo.toml:2]: "font-ir"? let the bikeshedding begin

File: ir/Cargo.toml (hunk 4 above)
Comment (rsheeter) [reply ir/Cargo.toml:7]: ty, will fix

**Pull request merged**